test/parallel/test-stream-writableState-uncorked-bufferedRequestCount.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const stream = require('stream');
+
+const writable = new stream.Writable();
+
+writable._write = (chunk, encoding, cb) => {
+  cb();
+};
+
+writable._writableState.corked = 1;
+writable._writableState.bufferedRequestCount = 1;
```

Comment:
can you please trigger the `corked` and `bufferedRequestCount` things using the normal API?